### DISCUSSION TITLE
How do you organize databases for different projects?

### DISCUSSION THREAD

**Discussion by dodyg** [geldata/gel#5812] [Q&A] [answered]
Say that I have two projects that needs two different databases.

- Project A
- Project B

How can I organize the databases?
- Now should I create one instance `my_instance` with two databases, `db_a` and `db_b` 
- Or two instances, `instance_a` and `instance_b`.

Because from what I know from the way `edgedb watch` works, it relies on default `edgedb` in an instance. So the first option isn't really practical.

**Comment by elprans** [comment, marked as answer]
Recent versions of the CLI and bindings support selecting a non-default database when initializing a project via the `--database` option, e.g `edgedb project init --database=my_project`.  Arguably this only makes sense for non-local instances, so for cloud instances this is the default behavior (the name of the database matches the name of the project directory).
